yarn-project/end-to-end/src/setup.ts

```diff
@@ -51,6 +60,10 @@ export async function setup(numberOfAccounts = 1): Promise<{
   config.unverifiedDataEmitterContract = deployL1ContractsValues.unverifiedDataEmitterAddress;
   config.inboxContract = deployL1ContractsValues.inboxAddress;
 
+  let archiver: Archiver | undefined;
+  if (createArchiver) {
```

Comment:
if we change test structure, we can set up an archiver independently and not change this file